pkg/master/master.go

```diff
@@ -215,6 +213,7 @@ func (c *Config) createNoneReconciler() reconcilers.EndpointReconciler {
 }
 
 func (c *Config) createLeaseReconciler() reconcilers.EndpointReconciler {
+	endpointClient := corev1client.NewForConfigOrDie(c.GenericConfig.LoopbackClientConfig)
```

Comment:
this is not the same as glog fataling

Reply:
Should we remedy this distinction in the klog refactor?